src/toggle-switch/toggle-switch.scss

```diff
@@ -92,10 +95,17 @@
   // #region Unchecked toggle switch
   background-color: var(--iui-color-background-1);
   border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  @media (forced-colors: active) {
+    border-color: CanvasText;
+  }
 
   &::after {
     background-color: var(--iui-color-foreground-body);
     opacity: var(--iui-opacity-2);
+    @media (forced-colors: active) {
+      background-color: CanvasText;
```

Comment:
Because we are changing the color of the handle between checked/unchecked states, the transition is messed up (in forced-colors mode). Not sure it's worth fixing but just something to be aware of

Reply:
It does seem to have a delay & no smooth transition.  I tried duplicating the `transition` within the `@media (forced-colors: active)` but that didn't help, so I'm not really sure how to fix this.  I find it weird that the transition breaks in toggle switches but not on side navigation.  Would be nice if we could fix it, but I would say let's proceed without it for now.